quick_tour/interacting_with_the_index.ipynb

```diff
@@ -121,7 +121,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    index.upsert(vectors=[("A", [0., 0.]), ])
</pre><p>let's change this to : index.upsert(vectors=[("A",[0.,0.])])  and remove the extra comma and space (, )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>